requirements.txt

```diff
@@ -50,6 +50,8 @@ idna==3.4
     # via requests
 imageio==2.31.1
     # via scikit-image
+importlib-resources==6.1.1
```

Comment:
Are you `pip-compile`ing with Python 3.10? I think this would not be installed in a 3.10 environment, which is what your `env.yml` creates.

It's a pretty small addition, though, so not a big deal.

Reply:
Yup, I'm on 3.10

https://github.com/nipreps/niworkflows/blob/0b2b12b94a88374ddf7ce2635b979a89c8b9130f/pyproject.toml#L27